api/src/main/java/com/messagebird/objects/AccessKey.java

```diff
@@ -1,9 +1,16 @@
 package com.messagebird.objects;
 
+import java.util.List;
+
 public class AccessKey {
     private String id;
-    private String key;
+    private String access_key;
```

Comment:
I think we should not use a snake case here also for `core_user_id, user_id and external_id`. We should move forward with a camel case. It applies to Getter/Setter methods as well. To fit in the proper JSON structure with the API, let's use `JsonProperty`. What do you think?